db/migrate/20160419214316_add_nuage_fields_to_cloud_subnets.rb

```diff
@@ -0,0 +1,8 @@
+class AddNuageFieldsToCloudSubnets < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_subnets, :enterprise_name, :string
+    add_column :cloud_subnets, :domain_id, :bigint
+    add_column :cloud_subnets, :zone_id, :bigint
+    add_column :cloud_subnets, :owner_id, :bigint
```

Comment:
Are the `bigint` fields here supposed to be references to tables inside the MIQ database?  Or, are these fields that would be populated directly from pulling inventory information from the VSD?

If these are collected from inventory, these should probably not be `bigint`s and instead be either `string`s or just `integer`s.  We typically use `bigint`s for primary key / foreign key values.

Also, I get the impression that these fields are being used as a composite key to look up a Nuage Subnet.  Is there a single UUID for a Nuage Subnet? Typically, we use the `ems_ref` field in the table to hold the single UUID referring back to the provider's subnet.